lib/octokit/client/deployments.rb

```diff
@@ -1,69 +1,87 @@
 module Octokit
   class Client
-
     # Methods for the Deployments API
     #
-    # @see https://developer.github.com/v3/repos/commits/deployments/
+    # @see https://developer.github.com/v3/repos/deployments/
     module Deployments
-
-      # Fetch a single deployment for a repository
+      # Get a single deployment
       #
       # @param repo [Integer, String, Repository, Hash] A GitHub repository
-      # @param deployment_id [Integer, String, Repository, Hash] A GitHub repository
+      # @param deployment_id [Integer] The ID of the deployment
       # @return <Sawyer::Resource> A single deployment
       # @see https://developer.github.com/v3/repos/deployments/#get-a-single-deployment
       def deployment(repo, deployment_id, options = {})
         get("#{Repository.path repo}/deployments/#{deployment_id}", options)
       end
 
-      # List all deployments for a repository
+      # List deployments
       #
       # @param repo [Integer, String, Repository, Hash] A GitHub repository
+      # @param options [String] :sha The SHA that was recorded at creation time.
+      # @param options [String] :ref The name of the ref. This can be a branch, tag, or SHA.
+      # @param options [String] :task The name of the task for the deployment (e.g., `deploy` or `deploy:migrations`).
+      # @param options [String] :environment The name of the environment that was deployed to (e.g., `staging` or `production`).
```

Comment:
One of the things that we sometimes do is list each of the optional params

```
@param options [type] :name_of_param $DESCRIPTION`
```

I really like that. A corollary is that if we have no optional params, we can leave off the `@param options` altogether.